scripts/test-rpc.sh

```
@@ -0,0 +1,622 @@
+    fi
+}
+
+JSON_DIR=""
+MANIFEST_FILE=""
+OUTPUT_MODE="pretty"
+RPC_URL=""
+CURRENT_CATEGORY=""
+CURRENT_CATEGORY_INDEX=-1
+RUN_TEST_LAST_RESPONSE=""
+
+ARGS=("$@")
```

Shellcheck (0.11.0)

[warning] ARGS appears unused. Verify use (or export if used externally).

(SC2034)